src/helpers/generateRandomName.ts

```diff
@@ -0,0 +1,16 @@
+import { Config, uniqueNamesGenerator } from 'unique-names-generator'
+import {
+  animalDictionary,
+  colorDictionary,
+} from '@/helpers/data/NameDictionary'
+
+export default function (address: string) {
+  const customConfig: Config = {
```

Comment:
You can put this object inside the function to get type annotations, so you don't need to defined `: Config`, which saves a little space

Reply:
This is how it's implemented by the library authors, no need to save space.